README.md

```diff
@@ -21,59 +22,95 @@ What's in the box?
 
 _Note for [asdf](https://github.com/asdf-vm/asdf) users - both Razor & the generated app have a `.tool-versions` file to help you get the right versions of things. You can change these - for example, erlang 19.3 is probably fine, but currently has installation problems on Macs._
 
 ## Installing & running
-* `mix escript.install https://github.com/carbonfive/razor/raw/master/razor_archives/razor`
-* Add the escript dir to your path or create a symlink, i.e. `ln -s /path/to/razor /usr/local/bin/`
-* Cut your app with `razor new YourAppName`
-* Or, if you want to install to a directory named differently from the app,
-* `razor new YourAppName your_target_dir`
+
+- `mix escript.install https://github.com/carbonfive/razor/raw/master/razor_archives/razor`
+- Add the escript dir to your path or create a symlink, i.e. `ln -s /path/to/razor /usr/local/bin/`
+- Cut your app with `razor new YourAppName`
+- Or, if you want to install to a directory named differently from the app,
+- `razor new YourAppName your_target_dir`
 
 
 ## Build your new project
-* `cd your_new_project_dir`
-* `mix deps.get` - install dependencies
-* `mix ecto.setup` - setup your local database
-* Prep js - `yarn install; brunch build`  
-If you have issues with this, see the "Before You Start" section above.
-* Create a `.env` file with variables MIX_ENV=dev and SECRET_KEY_BASE=your_secret_key_base.  
-Run `mix phoenix.gen.secret` to generate a good value for `SECRET_KEY_BASE`.
-* Set dev mode locally with `echo -n 'export MIX_ENV=dev' >> .env`
-* Verify the test suite passes with `mix test`
-* Run with heroku local/foreman, or load your secret key base from .env & run phoenix
-* `source .env; mix phoenix.server`
-* That's it! Visit your fully-featured app at `localhost:4000`.
+
+- `cd your_new_project_dir`
+- `mix deps.get` - install dependencies
+- `mix ecto.setup` - setup your local database
+- Prep js - `yarn install; brunch build` (If you have issues with this, see the "Before You Start" section above.)
+- Set your environment variables (see the "Configure your environment" section below)
+- Verify the test suite passes with `mix test`
+- Run your server (see the "Running the server" section below)
+- That's it! Visit your fully-featured app at `localhost:4000`.
 
 _Windows users - your executable may be `razor.bat` instead of `razor`_
 
+## Configure your environment
+
+To set environment variables locally for the `dev` environment, you can create a `.env` file & run the server with these environment variables (see "Running the server").
+
+## Running the server
+
+Your local server requires the environment variables in your `.env` file.
+
+- `heroku local` will read your `.env` and start the server locally, and has the benefit of similarities to your production environment.
+- `foreman` is another good option.
+- You can also do this on your own with `source .env` and `mix phoenix.server`
```

Comment:
Good idea.  Per our chat, we should change the instructions to inline the env vars, and just quickly suggest the foreman / heroku local alternatives.